<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>v_relationform</class>
 <widget class="QDialog" name="v_relationform">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Relation</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>381</width>
     <height>281</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="princpalLayout">
    <item>
     <widget class="QLabel" name="labelWindowTitle">
      <property name="font">
       <font>
        <pointsize>36</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Relation</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="labelTitle">
        <property name="text">
         <string>Title</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineTitle"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="labelDesc">
        <property name="text">
         <string>Description</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QTextEdit" name="textEdit"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="labelOriented">
        <property name="text">
         <string>Oriented ?</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <layout class="QHBoxLayout" name="btnLayout">
        <item>
         <widget class="QRadioButton" name="btnYes">
          <property name="text">
           <string>yes</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="btnNo">
          <property name="text">
           <string>No</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="btnCancel">
        <property name="text">
         <string>Cancel</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnSave">
        <property name="text">
         <string>Save</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
